README.md

```diff
@@ -1,3 +1,39 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Please kindly add a piece of code you find interesting or are proud of making. _Ex:_ I like the filter method because you can filter my original and also short name using the search bar, you can add this part or others as you like 😉👌
- [ ] [OPTIONAL] Of course its optional but I recommend using [Loom](https://www.loom.com/) to recorts use [Loom](https://www.loom.com/) to record your awesome guide video

Reply:
I will add it to the video!

I do want to say that I have an issue with loom in my computer. I can't make it work. That's why I'm using the other one.

Reply:
The new video can be found in the original pull request